### PR TITLE
working extension on anonymous server

### PR DESCRIPTION
**Pull request by echarles**
working extension on anonymous server

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,10 @@ install-dev:
 
 jlab-watch:
 	($(CONDA_ACTIVATE) ${ENV_NAME}; \
-		jupyter lab --watch --port 8234)
+		jupyter lab \
+			--watch \
+			--port 8234 \
+			--ServerApp.token=)
```
Comment (Carreau) [inline Makefile:53]: +1, should we remove that once tokens are supported ?

File: src/widget.tsx [hunk 2]
```diff
@@ -116,10 +116,10 @@ const KernelManagerComponent = (): JSX.Element => {
    */
 
   useEffect(() => {
-    const url = "http://localhost:8888/ks";
+    const url = "./ks";
```
Comment (Carreau) [inline src/widget.tsx:119]: ha ! Thanks ! I was using `document.location.origin+...` locally. I'm guessing this does not work if there is a prefix (like in hub), but ill be fixed once we actually use jlab services ?

Comment (echarles) [reply src/widget.tsx:119]: The correct fix will be for my next PR. I just wanted to get something a bit less harcoded. I will ping you on my next commit.

File: Makefile (hunk 1 above)
Comment (echarles) [reply Makefile:53]: Authentication will be enabled on my next commit.

File: src/widget.tsx (hunk 2 above)
Comment (Carreau) [reply src/widget.tsx:119]: Ok, no worries, cause just this one seem broken for me, and seem to fetch the URL for /lab, 

so maybe you meant `"/ks"` not `"./ks"` ?

**Pull request merged**